src/components/hosts/HostsTable.tsx

```diff
@@ -16,6 +16,7 @@ import {
   RowWrapper,
 } from '@patternfly/react-table';
 import { ExtraParamsType } from '@patternfly/react-table/dist/js/components/Table/base';
+import classnames from 'classnames';
```

Comment:
@jtomasek FYI PF has a utility function called `css` that uses this lib and does exactly the same :)